src/lddc.cpp

```diff
@@ -495,6 +501,71 @@ void Lddc::InitImuMsg(const ImuData& imu_data, ImuMsg& imu_msg, uint64_t& timest
   imu_msg.linear_acceleration.z = imu_data.acc_z;
 }
 
+void Lddc::StoreAccelerationValues(ImuMsg& imu_msg) {
+
+  _imu_accel_x_vector.push_back(imu_msg.linear_acceleration.x);

```

Comment:
Instead of `push_back` use `emplace_back` for a better performance use

https://www.geeksforgeeks.org/push_back-vs-emplace_back-in-cpp-stl-vectors/#:~:text=Everytime%20push_back%20a%20value%20to%20a%20vector%20%2C%20cpp%20will%20create%20a%20new%20vector%20which%20copied%20the%20new%20value%20from%20parameter%20and%20all%20values%20in%20the%20old%20vector%C2%A0